tests/multibackend_test.py

```diff
@@ -48,7 +48,7 @@ class MultiBackendTest(jtu.JaxTestCase):
   def testMultiBackend(self, backend):
     if backend not in ('cpu', jtu.device_under_test(), None):
       raise SkipTest()
-    @partial(api.jit, backend=backend)
```

Comment:
Instead of doing this, I would rewrite (or eliminate) the test. It is not clear that in the absence of the backend argument this test adds value.